.github/PULL_REQUEST_TEMPLATE/feature.md

```diff
@@ -0,0 +1,15 @@
+(Make sure your title is either: 'fix', 'chore', or 'feat' then your title. ex: `fix: add new plugin`)
```

Comment:
Do you think it's worth linking the conventional commits page here?

Is it worth also worth linking the contributing.md or website page?

Reply:
Yeah I was thinking about that and was considering moving this into contributing.md, and then just simply linking to contributing.md instead